packages/shared/src/broker.ts

```diff
@@ -55,12 +55,15 @@ const transformedDcaParamsSchema = dcaParamsSchema.transform(
   }),
 );
 
-const transformedCcmParamsSchema = <T extends HexString | undefined>(defaultValue: T) =>
-  ccmParamsSchema.transform(({ message, gasBudget, cfParameters }) => ({
+const transformedCcmParamsSchema = ccmParamsSchema.transform(
+  ({ message, gasBudget, additionalData }) => ({
     message,
     gas_budget: gasBudget,
-    cf_parameters: cfParameters ?? defaultValue,
-  }));
+    ccm_additional_data: additionalData,
+    /** @deprecated DEPRECATED(1.8) we still need to pass cf_parameters until 1.8 is deployed to all networks */
+    cf_parameters: additionalData,
```

Comment:
tested against the backspin broker (1.8) and the persa broker (1.7). passing both keys for now seems to do the trick